pkg/defaulting/images_test.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2012 Datadog, Inc.
+
+package defaulting
+
+import "testing"
+
+func TestGetLatestAgentImage(t *testing.T) {
+	tests := []struct {
+		name string
+		want string
+	}{
+		{
+			name: "default",
+			want: "gcr.io/datadoghq/agent:7.30.0",
```

Comment:
This test will fail every time the latest version is updated. Should we change the test to match a regex instead of a specific version to avoid that?

Reply:
I initialy didn't want to commit this file :)

Reply:
test updated to support futur tag bump